File: panorama/src/main/layout/kotlin/com/isycat/dotaaddon/panorama/unitCardWrapperCardPanel2.kt
package com.isycat.dotaaddon.panorama

import com.isycat.dota.panorama.Button
import com.isycat.dota.panorama.Label
import com.isycat.dota.panorama.Panel

fun unitCardWrapperCardPanel2() =
    Panel(classes = "UnitCardZXCTWO", hittest = false) {
        Panel(classes = "CardTransformWrapper", id = "CardTransformWrapperX", hittest = false) {
            Label(id = "CardQuantity", hittest = false)
            Button(classes = "ShopItem", id = "ShopItemX")
        }
    }

File: content/panorama/scripts/custom_game/unitCardWrapperCardPanel2.js
// package: com.isycat.dotaaddon.panorama

function unitCardWrapperCardPanel2() {
    return new Panel("UnitCardZXCTWO", false, () => {
        return new Panel("CardTransformWrapper", "CardTransformWrapperX", false, () => {
            new Label("CardQuantity", false);
            return new Button("ShopItem", "ShopItemX");
        });
    });
}

